atom-notebook install failing at jupyter-js-utils

See this issue on atom-notebook: https://github.com/jupyter/atom-notebook/issues/32#issuecomment-184922403

`npm ERR! Failed at the jupyter-js-utils@0.2.17 postinstall script 'npm dedupe'.`

Atom uses node v0.10.40 and npm v2.13.3 to install Node packages.

Any idea what may be causing this?

--- a/package.json
+++ b/package.json
@@ -23,7 +23,6 @@
     "build:test": "tsc --project test/src",
     "build": "npm run build:src && npm run build:test",
     "docs": "typedoc --options scripts/tdoptions.json",
-    "postinstall": "npm dedupe",
     "prepublish": "npm run build",
     "test:coverage": "cd test && istanbul cover _mocha -- build/index.js --foo bar",
     "test": "mocha test/build/index.js --foo bar"

